modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,430 @@
+
+* Before this update, if a YAML manifest that a pipeline run uses (for example, the manifest of a task or pipeline) had syntax errors, the logged error message was unspecific or no error message was logged. With this update, the logged error message includes the syntax errors.
+
+* Before this update, when you used the `buildah` cluster task with a secret with the `.dockerconfigjson` file provided using a workspace, the task failed during the `cp` command because the `/root/.docker` directory did not exist. With this update, the task completes successfully.
+
+* Before this update, if a pipeline run timed out and a `TaskRun` or `CustomRun` resource that this pipeline run included was deleted, the pipeline run execution was blocked and never completed. With this update, the execution correctly ends, logging a canceled state.
+
+* Before this update, when using a resolver to incorporate a task from a remote source, the resolver automatically added the `kind` value of `Task` to the resulting  specification. With this update, the resolver does not add a `kind` value to the specification.
+
+* Before this update, when you set configuration options using an `options:` section in the `TektonConfig` CR, these options were sometimes not applied correctly. With this update, the options are applied correctly.
+
+* Before this update, if you set the `enable-api-fields` field and certain other fields in the `TektonConfig` CR, the settings were lost after any update of {pipelines-shortname}. With this update, the settings are preserved during updates.
+
+* Before this update, if you configured the Horizontal Pod Autoscaler (HPA) using the options section in the `TektonConfig` CR, any existing HPA was updated correctly but a new HPA was not created when required. With this update, Horizontal Pod Autoscaler configuration using the options section works correctly.
+
+* Before this update, you could erroneously change the `targetNamespace` field in the `TektonConfig` CR, creating an unsupported configuration. With this update, you can no longer change this field. Changing the target namespace name from `openshift-pipelines` is not supported.
+
+
+* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} operator controller would fail to create default resources in new namespaces correctly. With this update, the controller functions correctly.
```

Comment:
```suggestion
* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} Operator controller would fail to create default resources in new namespaces correctly. With this update, the controller functions correctly.
```